app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class='container'>
+  
+  <div class='form_container'>
+  <h2>Sign up</h2>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    
+    <div class="field">
+    <%= f.label :fullName %><br />
+    <%= f.text_field :name, autofocus: true %>
+  </div>
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
```

Comment:
- [ ] When I add the password_confirmation, I am able to sucessfully sign up. From the server logs in the terminal, I sucessfully sign up and receive a link to confirm my email, but that is not displayed on the browser. The browser, I am redirected to the Sign in page
![image](https://github.com/souad988/budget_tracker_app/assets/62849872/d03334b5-f4ae-40da-8ab0-8d4e7530293c)
However, the message you see in the terminal does not necessarily reflect what is displayed in the browser. To display the confirmation message in the browser, you need to handle the response from the server and render an appropriate view or redirect the user to a specific page.